src/_settings.scss

```diff
@@ -220,7 +221,15 @@ $shell-g-spacing-6x-large: ceil($shell-g-spacing-base / 2 * 12) !default;
 
 
 
-/* 5. Breakpoints (Global)
+/* 5. Border Radius (Global)
```

Comment:
I think it makes sense to categorise this for scalability like all the other sections. Maybe something like **Cosmetics**?